modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1693,6 +1693,10 @@
 			<category name="Upgrade" />
 			<description name="Replace constructors that use the empty constructor of the SchedulerEntryImpl class." />
 		</check>
+		<check name="UpgradeJavaScreenContributorClassCheck">
+			<category name="Upgrade" />
+			<description name="Replace class 'PortalSettingsConfigurationScreenContributor' by 'ConfigurationScreenWrapper' and create an inner class'" />
```

Comment:
@nicolas-sss, there is an extra single quotes at the end of this message, can you please remove it?